### PR TITLE
Create Setup server on localhost.md

### PR DESCRIPTION
**Pull request by Orange-hanter**

### CODE REVIEW

File: Docs/Setup server on localhost.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Working with pgAdmin:
+1. Go to pgAdmin after copying the database code.
+ it can be found here:
+ https://github.com/f1nal3/Juniorgram/blob/master/Utility/SQL/Database_Scheme.sql
+2. Right-click on Databases and then Create / Database ..
+3. Create a table with an arbitrary name.
```
Comment (f1nal3) [inline Docs/Setup server on localhost.md:6]: A table or a database?

File: Docs/Setup server on localhost.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+"dbname = your_bdname user = postgres hostaddr = 127.0.0.1 port = 5432 password = your_password"
+
+3. Then build the project.
+
+
+## Work in the console:
+1. Go to the \ Juniorgram \ Server \ bin \ Server \ Debug folder and run cmd
+2. Run the command Server.Exec.exe -p 65001
+3. Start the client.
+4. Done, the server is running locally.
```
Comment (f1nal3) [inline Docs/Setup server on localhost.md:30]: The server is running locally after part 2 =)

File: Docs/Setup server on localhost.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ it can be found here:
+ https://github.com/f1nal3/Juniorgram/blob/master/Utility/SQL/Database_Scheme.sql
+2. Right-click on Databases and then Create / Database ..
+3. Create a table with an arbitrary name.
+4. Next, click on the created base and select the Query Tool
+5. Paste the previously copied code and run F5
+
+
+## Work in the editor:
+1. In ServerInfo.cpp, specify the address of the local host
```
Comment (f1nal3) [inline Docs/Setup server on localhost.md:12]: I'd separate this from the rest cause this really concerns Client and has nothing to do with the server. Can be transferred to the last part with all the client parts there

File: Docs/Setup server on localhost.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+   
+    ```c++
+      constexpr std :: string_view address = "127.0.0.1"
```
Comment (f1nal3) [inline Docs/Setup server on localhost.md:15]: inconsistent with line 17 std::uint16_t and here std :: string view.

File: Docs/Setup server on localhost.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      //and
+      constexpr std::uint16_t port = test_server;
+    ```
+
+2. Change the parameters in DataBaseOptions.hpp
+"dbname = your_bdname user = postgres hostaddr = 127.0.0.1 port = 5432 password = your_password"
```
Comment (f1nal3) [inline Docs/Setup server on localhost.md:21]: this is also c++ code :)

**Review by f1nal3 — CHANGES_REQUESTED** [review]

**Review by f1nal3 — APPROVED** [review]

**Review by StasDash — APPROVED** [review]

**Pull request merged**